src/neo/Ledger/GlobalVariableState.cs

```diff
@@ -0,0 +1,41 @@
+using Neo.IO;
+using Neo.Network.P2P.Payloads;
+using System.IO;
+
+namespace Neo.Ledger
+{
+    public class GlobalVariableState : ICloneable<GlobalVariableState>, ISerializable
+    {
+        public uint GlobalCounter = 0x00000000;
```

Comment:
Will it be better to  rename the class `ContractIdState`,  and the variable `Id` ?